pyomo/neos/tests/test_neos.py

```diff
@@ -90,6 +88,177 @@ def test_kestrel_plugin(self):
         self.assertEqual(results.solver[0].status, pyo.SolverStatus.ok)
         self.assertAlmostEqual(pyo.value(m.x), 0.5)
 
+    def test_doc(self):
+        kestrel = kestrelAMPL()
+        tmp = [tuple(name.split(':')) for name in kestrel.solvers()]
+        amplsolvers = set(v[0].lower() for v in tmp if v[1]=='AMPL')
+
+        doc = pyomo.neos.doc
+        dockeys = set(doc.keys())
+
+        self.assertEqual(amplsolvers, dockeys)
+
+        #gamssolvers = set(v[0].lower() for v in tmp if v[1]=='GAMS')
```

Comment:
These lines are commented out - do we still need them?

Reply:
Please leave these to document the test change we should consider later.